staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_external.go

```diff
@@ -51,6 +54,19 @@ const (
 // new load balancers and updating existing load balancers, recognizing when
 // each is needed.
 func (g *Cloud) ensureExternalLoadBalancer(clusterName string, clusterID string, apiService *v1.Service, existingFwdRule *compute.ForwardingRule, nodes []*v1.Node) (*v1.LoadBalancerStatus, error) {
+	// Skip service handling if managed by ingress-gce using Regional Backend Services
+	if val, ok := apiService.Annotations[RBSAnnotationKey]; ok && val == RBSEnabled {
+		return nil, cloudprovider.ImplementedElsewhere
+	}
+	// Skip service handling if service has Regional Backend Services finalizer
+	if hasFinalizer(apiService, ELBRbsFinalizer) {
+		return nil, cloudprovider.ImplementedElsewhere
+	}
+	// Skip service handling if it has Regional Backend Service created by Ingress-GCE
+	if existingFwdRule.BackendService != "" {
```

Comment:
you need to check if fwd rule exist
if existingFwdRule != nil &&  existingFwdRule.BackendService != ""

Any tests for this case?